number_theory/modular.h

```diff
@@ -132,6 +134,19 @@ class Modular {
   }
 };
 
+// Returns the modular inverse of |number| in modulo |mod| if exists.
+// Otherwise, throws an invalid_argument exception.
+template <typename T,
+          std::enable_if_t<std::numeric_limits<T>::is_integer, bool> = true>
```

Comment:
I think we only need the `std::enable_if` trick when we want to distinguish overloaded templates. Otherwise, we use `static_assert`.

Reply:
I think I was using `modular_internal::normalize` as an example. Should we change it to `static_assert` as well?

Reply:
Yeah we need to change it.